WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -369,6 +392,10 @@ import CoreData
                 if slide.evaluated == false && yirConfig.personalizedSlides.donateCount.isEnabled {
                     needsDonatingPopulation = true
                 }
+            case WMFYearInReviewPersonalizedSlideID.savedCount.rawValue:
+                if slide.evaluated == false && yirConfig.personalizedSlides.saveCount.isEnabled && username != nil {
```

Comment:
Update: ok, I see now that these personalized slides are only shown if they are logged in. I will work on adding this logic to https://github.com/wikimedia/wikipedia-ios/pull/5085.

Reply:
Update again: I removed this username check, since technically the saved article data is available to us whether they are logged in or not, whereas with the edit counts we must have a username before we can populate personalized data.

So the local personalized data will still be populated even if logged out, but we will hide the personalized slides at the entry point / coordinator level. I will double-check this when I wrap up https://github.com/wikimedia/wikipedia-ios/pull/5085.

I checked with Product and this is acceptable.